internal/sign/sign.go

```diff
@@ -0,0 +1,47 @@
+// Copyright 2023 The Falco Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sign
```

Comment:
nit: maybe call this verify, this is not signing anything and it is just verifying the signatures

Reply:
counter-offer, I can call it `signature` because it may be used to contain some additional helper functions to deal with signatures :)

Reply:
sgtm :D